### ISSUE TITLE
Appium 1.5 doctor display xcode info in Win

### ISSUE THREAD

**Issue by monkeytest15** [appium/appium-doctor#12]
I have already update appium to 1.5

In win10, run `appium-doctor` in terminal and display `appium-doctor` command not found.

Then `npm install appium-doctor -g` and run again, but display
```shell
info AppiumDoctor ### Diagnostic starting ###
WARN AppiumDoctor x Xcode is Not installed!
```

but my platform is Win10.

**Comment by imurchie** [comment]
Thank you for bringing this to our attention.

**Comment by imurchie** [comment]
This is fixed in version 1.2.0. Can you update and rerun?
